_includes/_companies.html

```diff
@@ -76,6 +76,30 @@
       </div>
 
       <div>
+         <a href="http://oxx.co.id">
+            <image src="images/logo_oxx.png">
+         </a>
+         <h3>Oxx</h3>
+         <p>Teknologi untuk kebutuhkan bisnis dan organisasi.</p>
+      </div>
+
+      <div>
+         <a href="http://taxies.co.id">
+            <image src="images/logo_taxies.svg">
+         </a>
+         <h3>Taxies</h3>
+         <p>Taxies adalah sebuah layanan perangkat lunak untuk manajemen perusahaan taksi.</p>
+      </div>
+
+      <div>
+         <a href="http://unitedninja.id">
```

Comment:
we found out that the page here https://unitedninja.id/tentang-kami is using lorem ipsum, please use a proper company description (like address, the history, phone numbers, etc)